### PR TITLE
Soarian r4 condition Resource Documentation

### PR DESCRIPTION
**Pull request by VamsiMure**
Description
Soarian FHIR R4 Condition  resource

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
condition-1
condition-2
condition-3
condition-4
condition-5
condition-6
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/soarian/r4/clinical/summary/condition.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+---------------|----------------------------------------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------
+ `_id`         | This, or `patient`                           | [`token`](https://hl7.org/fhir/R4/search.html#token)        | The logical resource id associated with the resource.
+ `patient`	   | This, or `_id`					              | [`reference`](https://hl7.org/fhir/r4/search.html#reference)| The patient who has the condition.
+ `encounter`   | No 										  | [`reference`](https://hl7.org/fhir/r4/search.html#reference)| The encounter associated with the condition.
+ `status`      | No 										  | [`token`](https://hl7.org/fhir/R4/search.html#token)        | The status of the condition. May be a list separated by commas.
+ `_revinclude` | No 										  | [`_revinclude`](https://hl7.org/fhir/search.html#revinclude)| A request to include any Provenance resource in the bundle that refers to condition resource in the search results. Only supported with Provenance.
```
Comment (HRNWriter) [inline content/soarian/r4/clinical/summary/condition.md:50]: Please add "the" before "condition resource", so the sentence reads as follows:
A request to include any Provenance resource in the bundle that refers to the condition resource in the search results.

Comment (VamsiMure) [reply content/soarian/r4/clinical/summary/condition.md]: Done. File has been updated.

**Review by slvsridhar — APPROVED** [review]

**Comment by sonikrish** [comment]
Screenshot 2021-10-18 at 10 50 33 AM
Authorization type should be given under each example section. It should not be given as a separate heading.

File: content/soarian/r4/clinical/summary/condition.md (hunk 1 above)
Comment (HRNWriter) [reply content/soarian/r4/clinical/summary/condition.md]: Looks good, thank you.

**Review by HRNWriter — APPROVED** [review]
+1

**Pull request merged**